### DISCUSSION TITLE
errors when adding and editing pages. It says "the index wiki does not exist"

### DISCUSSION THREAD

**Discussion by SMorton87** [requarks/wiki#6722] [Error / Bug Report]
I am receiving an error when editing our company's page ( something that JUST broke because we have been able to do it fine before) I also get this message after I receive that error (see below) we are not sure what could be causing this, any suggestions?

![image](https://github.com/requarks/wiki/assets/144940254/498dd6fd-ac04-493c-9104-b0f4b00afb6a)

**Comment by NGPixel** [comment]
What error message?
